### ISSUE TITLE
Options for have incremental volume (fan speed)?

### ISSUE THREAD

**Issue by cyrusauyeungc** [DoctorNSA/homebridge-musiccast-tv#21]
The plugin is working perfectly on my Yamaha WXA-50 with the ability to adjust input and volume (as fan speed).
I am not sure if this is the right place to ask but would there be any settings/shortcuts other than Nested Ifs to control the volume?
I have paired a Philips Hue remote that comes with On/Off/Brightness Up/Down and I'd like to use the brightness buttons as the volume control button.
Would you be able to shed some lights on it? thanks and appreciate your effort!

**Comment by aangert** [comment]
Yes! There is a TelevisionSpeakerService, that supports both increasing/decreasing the volume by one and directly setting it to a specific value (between 1 and 100). Unfortunately the Apple Home app doesn't support directly setting it to a specific value, which is why I used the fan speed workaround. 
Depending on which app you use setting it to a specific value should be supported

**Comment by cyrusauyeungc** [comment]
thanks for the prompt reply and appreciate that!

I'll definitely look in to the service that you've mentioned. 
Yea it's a glitch that Apple Home doesn't support fan speed/brightness to a specific value,
Originally I tried to pull the volume (fan speed) from the "Get State of a certain accessory", I tried to do a +5 but I am unable to feed the x+5 fan speed into the amplifier.... thanks for confirming this though :)
